src/components/AppStructure.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PureComponent from 'react-pure-render/component';
+import Header from './Header.jsx';
```

Comment:
Nit: Add an empty new line above this line to separate the plugin imports from imports that are related to functions/classes that we defined. This is also in the style guide.